specs/SIM/SODA_InfrastructureManagerDesign.md

```diff
@@ -0,0 +1,257 @@
+﻿
+# SODA Infrastructure Manager (delfin) Design Document
+**Authors:** [Najmudheen CT](https://github.com/NajmudheenCT), [Sanil Kumar D](https://github.com/skdwriting), [Xulin](https://github.com/wisererik),  [Ashit Kumar](https://github.com/kumarashit)
+
+This documentation serves as the design spec for SODA Infrastructure Management
+## Goal
+To build an open source infrastructure management platform for heterogeneous storage environments. 
+* Framework for heterogeneous resource management 
+* APIS for storage resource monitoring and alerts
+* Common storage resource models
+
+## Non-Goals
+* GUI for infrastructure management.
+* Performance metric models
+* Switch and Host discover requirements
+* Switch and Host models
+* Resource configuration requirements and design
+* deployment model
+
+## Assumptions and Constraints
+NA
+## Motivation and background
+Today’s IT infrastructure spans across multi vendor devices and technologies. Administration of heterogeneous platforms is a challenge to storage admins.  Vendor supplied management software is either vendor locked in or not open source.  SODA foundation has vision to standardize and bring heterogeneous device management using a single platform.
+
+## Requirement Analysis
+
+One single pane of glass for managing heterogeneous storage devices, switch and host . A standard API model to manage devices from different vendors and a single source to collect monitoring and alert information from heterogeneous platforms.  
+### Storage Discovery Requirements
+#### Storage Discovery:-
+User should be able to add supported device models to the system for discovery
+#### Storage Monitoring and Alerts: 
+Get basic storage information, hardware information, logical resources information, alert information , performance information  and configure resources and services in the storage device. 
+
+### Feature Requirements
+#### Requirement Analysis
+Analyzed Open Source monitoring solution.
+
+1.	Stor2rrd : [Analysis](https://drive.google.com/open?id=1yJn6wcji24NvObbrxvfSVzmo6--QGSI_v5P8n4Kcc8M),
```

Comment:
Can we move it to some other place? Drive may not be accessible for all

Reply:
I hope we can remove this link.. there is no relevant  information to support this design.

Reply:
@kumarashit Need your suggestion on this

Reply:
Yes, please remove this link. IMO, just mention it and we can give the link of the homepage

Reply:
done

Reply:
Ok thanks